pkgs/by-name/ca/cables/package.nix

```
@@ -0,0 +1,38 @@
+{ pkgs, lib, ... }:
+
+let
+  pname = "cables";
+  version = "0.3.2";
+  name = "${pname}-${version}";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/cables-gl/cables_electron/releases/download/v0.3.2/cables-0.3.2-linux-x64.AppImage";
+    sha256 = "sha256-D5Bgg5D03FPQF2HKow4yehcqToo4dpPudBg0W4UnqFs=";
+  };
+
+  appimageContents = pkgs.appimageTools.extract {
```

Try to avoid referencing `pkgs` directly. It makes it harder to call `.override` on the package. The dots should be superfluous.

```suggestion
{ 
  lib,
  fetchurl,
  appimageTools
}:

let
  pname = "cables";
  version = "0.3.2";
  name = "${pname}-${version}";

  src = fetchurl {
    url = "https://github.com/cables-gl/cables_electron/releases/download/v0.3.2/cables-0.3.2-linux-x64.AppImage";
    sha256 = "sha256-D5Bgg5D03FPQF2HKow4yehcqToo4dpPudBg0W4UnqFs=";
  };

  appimageContents = appimageTools.extract {
```